src/CodeGen_C.cpp

```diff
@@ -155,8 +155,9 @@ template<typename T>
 inline int halide_popcount(T a) {
     int bits_set = 0;
     while (a != 0) {
```

Comment:
FWIW, ~all modern compilers should have popcount as an intrinsic -- see `popcount64` in Util.cpp, maybe we should just adapt that here